torchaudio/transforms.py

```diff
@@ -640,6 +641,83 @@ def forward(self, complex_specgrams, overriding_rate=None):
         return F.phase_vocoder(complex_specgrams, rate, self.phase_advance)
 
 
+class Fade(torch.nn.Module):
+    r"""Add a fade in and/or fade out to an waveform.
+
+    Args:
+        fade_in_len (int, optional): Length of fade-in (time frames). (Default: ``0``)
+        fade_out_len (int, optional): Length of fade-out (time frames). (Default: ``0``)
+        fade_shape (str, optional): Shape of fade. Must be one of:
```

Comment:
Where does this notation come from?

Reply:
The notions come from https://pysox.readthedocs.io/en/latest/api.html#sox.transform.Transformer.fade and looking into fade in http://sox.sourceforge.net/sox.html they are also coherent.

'p' for inverted parabola are not working properly using Sox so I opted to follow Audacity nomenclature using 'e' for exponential fade instead.

Reply:
We don't have to use the exact same notation. How about `quarter_sine`, `half_sine`, `linear`, `logarithmic`, and `exponential`?

Reply:
Updated.